tests/lib/linter.js

```diff
@@ -469,176 +466,182 @@ describe("Linter", () => {
         const code = "function foo() { q: for(;;) { break q; } } function bar () { var q = t; } var baz = (() => { return 1; });";
 
         it("should retrieve the global scope correctly from a Program", () => {
-            const config = { rules: {} };
-
-            linter.reset();
-            linter.on("Program", () => {
+            const config = { rules: { checker: "error" }, parserOptions: { ecmaVersion: 6 } };
```

Comment:
Most of the tests in this suite had previously been broken, because `parserOptions: { ecmaVersion: 6 }` wasn't getting set, resulting in a parser error. This caused `Linter` to not fire any rule listeners, so no assertions failed because all the assertions were in a rule listener.

This isn't a problem anymore after this PR because the tests now assert that the rule listeners fired.